_posts/2021-06-05-memn

```diff
@@ -0,0 +1,92 @@
+
+Bevor wir mit dem Downloaden von unseren Software Packages starten können, richten wir das virtuelle private Netzwerk (VPN) ein. Das Cisco AnyConnect VPN kann eine sichere Verbindung mittels einem verschlüsselten Tunnel zwischen deinem Laptop und dem Internet herstellen. Es gibt vielerelei Nutzen vom VPN. In erster Linie hast du einen sog. Fernzugriff, auch Remote-Access genannt, und bekommst Anschluss an das lokale Netzwerk deiner Schule. Dies ermöglichst dir Zugriff auf private Ressourcen oder auf wichtige Tools. Das VPN bietet dir gleichzeitg Datenschutz und Sicherheit vor Hackern. Die Anleitung für die Installation des VPN Client für MacUser findest du auf der [ZHAW Service Desk - Website](https://servicedesk.zhaw.ch/tas/public/ssp/content/detail/knowledgeitem?unid=4f3b1c57c259455d96ee3189ee8d1219). 
+
+### Webbrowser:
+
+In Macbook gilt Safari als der eingebaute, schnelle Standard Webbrowser. Es gibt Dienste, wie das Intranet der ZHAW, Youtube TV welche nicht auf Safari funktionieren und andere Browser verwendet werden. Ich empfehle dir dabei diese zwei Webbrowser zu installieren.
+
+- **Google Chrome**: ist der beliebteste Browser und bietet durch seine vielen leistungsstarken Erweiterungen eine grosse Flexibilität und unterstützt zahlreiche Webanwendungen und Dienste.
+- **Mozilla Firefox**: konzentriert sich darauf, Tracker zu blockieren und datenschutzfreundliche Funktionen anzubieten. 
+
+### Arbeitsumgebung und Produktivität
+
+Der MacBook ist mit den iWork-Programmen Pages, Numbers und Keynote gut ausgestattet. Die Produktivität oder die Effizienz im Team steigern mittels folgenden Apps: 
+
+- **Adobe Acrobat Reader DC**: ermöglichst das Erstellen, Bearbeiten und Unterzeichnen von PDF-Dokumenten.
+- **Microsoft Office Education**: (vormals Office 365) ist eine Standardbürosoftware für das Studium, welche aus Excel (Tabellenkalkulation), OneNote (digitale Notizbücher), Outlook (E-Mail-Client mit Kalender und Aufgabenverwaltung), PowerPoint (Präsentation) Teams (Kommunikation), Visio (Diagramme), und Word (Textverarbeitung) besteht.
+- **OmniGraffle**: ist eine Möglichkeit die Organisation und Aufbereitung von Daten in übersichtliche Diagramme, Charts und Mindmaps zu illustrieren.
+- **Draw.io**: ist eine kostenlose Online-Diagrammsoftware zur Erstellung von Flussdiagrammen, Prozessdiagrammen, Organigrammen, und Netzwerkdiagrammen.
+
+### Kommunikation
+
+Für das Home Office und Remote Learning benutzen wir folgende Programme: 
+
+- **Cisco Webex Meetings**: ist ein Anbieter von Videokonferenzen, IP-Telefonie, Instant-Messaging, Dateiübertragung und Screen-Sharing.
+- **Zoom.us**: bietet Remote-Konferenzdienste, die Videokonferenzen, Online-Besprechungen, Chat und mobile Zusammenarbeit kombinieren
+
+### Entwicklungstools
+
+Damit du startklar für die Informatik-Kurse bist, brauchst du diese Programme:
+
+-**Git / GitHub**: ist ein freies verteiltes Versionskontrollsystem, um deine Arbeiten schnell und effizient zu bearbeiten. 
```

Comment:
Wichtig: In Markdown musst du einen Leerschlag nach dem Minus machen, damit du eine Liste erzeugst.

Nach einem Doppelpunkt schreibt man gross weiter ;)

```suggestion
- **Git / GitHub**: Ist ein freies verteiltes Versionskontrollsystem, um deine Arbeiten schnell und effizient zu sichern.
```